src/main/java/mediathek/filmlisten/FilmeImportierenDateiThread.java

```diff
@@ -26,4 +35,11 @@ public void run() {
         final boolean result = downloadAction.performDownload(pfad, listeFilme, days);
         onFinishedAction.onFinished(result);
     }
+    
+    public static String getDirnameFromPathname(String pathname)
+    {
+        Path path = Paths.get(pathname);
+        String dirname = path.getRoot().toString()+path.subpath(0, path.getNameCount()-1).toString();
```

Comment:
WTF? If I don't missed anything this would convert "\MyCloud\Public\Shared Videos\RBB\Abo\Chez Krömer" to "\MyCloud\Public\Shared Videos\RBB\Abo\" so just the path of the parent element. So why not just `path.getParent()`?

Reply:
@Nicklas2751 
Previous: I don't know the Backend process in sufficient details.
So see this example and feel free to fix it to your solution. This is one way.
No reason to become angry (read from comment fromulation)

Reply:
@derreisende77 
Here you get some facts with examples :) Unfortunately I cannot test that, unless someone tells me how to call that part.

Reply:
@derreisende77 
Ja, getestet mit lokal und smb share, dank stumpfer pfade ist da aber kein Unterschied. Woher kriege ich da den abo-ordner-pfad? -> neuer pull request